lib/util/Ranges.v3

```diff
@@ -78,38 +78,38 @@ component Ranges {
 	}
 	// Performs binary search and returns the index.
 	def binarySearch<T>(range: Range<T>, val: T, lt: (T, T) -> bool) -> int {
```

Comment:
I think we should name these three methods `binarySearchEq`, `binarySearchLt`, and `binarySearchGt` and comment them uniformly. See below.